csatar-theme/layouts/default.htm

```diff
@@ -0,0 +1,31 @@
+description = "Default layout"
+
+[cookieBanner cookiebannerOverRide]
+include_css = 1
+cookie_manager_page = "index"
+==
+<!DOCTYPE html>
+<html class='h-100'>
+    <head>
+        {% partial 'site/meta' %}
+    </head>
+    <body class="{{ 'page-'~this.page.id }} h-100 d-flex flex-column">
+
+        <!-- Header -->
+        <header id="layout-header">
+            {% partial 'site/header' %}
+        </header>
+
+        <!-- Content -->
+        <section id="layout-content" class="flex-grow-1">
+            {% page %}
+        </section>
+
+        <!-- Footer -->
+        <footer id="layout-footer">
+            {% partial 'site/footer' %}
+            {% component 'cookiebannerOverRide' %}
+        </footer>
```

Comment:
The readme of the theme states:

**Important**: Make sure you keep the `{% styles %}` and `{% scripts %}` placeholder tags as these are used by plugins for injecting assets.

Reply:
Those tags are in the meta.htm partial.